#goal definition
std_msgs/Empty goal
---
# Define the result
bool spawn_success
---
# Define a feedback message
bool spawn_success
